modules/internal/ChapelLocale.chpl

```diff
@@ -226,6 +226,14 @@ module ChapelLocale {
   */
   inline proc locale.maxTaskPar: int { return this._value.maxTaskPar; }
 
+  /*
+    Get the number of colocales on the locale's node.
+
+    :returns: the number of colocales on the locale's node
+    :rtype: int
```

Comment:
Can you add some of the details from the PR message here? Particularly the example of `-nl 1` and `-nl 1x2`

Also a back-reference to some of the colocale docs might be nice

Reply:
Done.